src/plugins/data/public/ui/typeahead/suggestion_component.tsx

```diff
@@ -46,6 +46,7 @@ interface Props {
   suggestion: QuerySuggestion;
   innerRef: (node: HTMLDivElement) => void;
   ariaId: string;
+  shouldDisplayDescription: boolean;
```

Comment:
Can you please add docs for this parameter (and potentially the others, if you don't mind)?

I think that it's worth mentioning this is an  automatically determined argument, calculated from the width of the component.